test/e2e/cron_test.go

```diff
@@ -64,10 +64,7 @@ spec:
     templates:
       - name: whalesay
         container:
-          image: python:alpine3.6
-          imagePullPolicy: IfNotPresent
-          command: ["sh", -c]
-          args: ["echo hello"]`).
+          image: argoproj/argosay:v2`).
```

Comment:
Just curious - what's wrong with the existing image?

Reply:
We preload the argosay image in our CI, so we don't need to download it